pkg/cmd/roachtest/tests/allocator.go

```diff
@@ -81,18 +82,10 @@ func registerAllocator(r registry.Registry) {
 		c.Start(ctx, t.L(), startOpts, install.MakeClusterSettings(), c.Range(start+1, nodes))
 		c.Run(ctx, option.WithNodes(c.Node(1)), "./cockroach workload init kv --drop {pgurl:1}")
 		for node := 1; node <= nodes; node++ {
-			node := node
-			// TODO(dan): Ideally, the test would fail if this queryload failed,
-			// but we can't put it in monitor as-is because the test deadlocks.
-			go func() {
+			t.Go(func(taskCtx context.Context, _ *logger.Logger) error {
 				cmd := fmt.Sprintf("./cockroach workload run kv --tolerate-errors --min-block-bytes=8 --max-block-bytes=127 {pgurl%s}", c.Node(node))
-				l, err := t.L().ChildLogger(fmt.Sprintf(`kv-%d`, node))
```

Comment:
Looks like this logger was previously unused?

Reply:
Yep, had to look twice before removing, and in actual fact was dead code.